src/Analysis.R

```diff
@@ -60,25 +51,33 @@ penguins_data <- penguins_data |> drop_na()
 
 ## ----descriptives----
 
-# NOTE: Descriptives not computed, as `gtsummary` package is too heavy
+## Simple summary:
+penguins_data |> summary()
 
-## Create descriptive statistics table:
-# descriptive_table <- penguins_data |> tbl_summary()
-# descriptive_table
-## TODO: Uncomment if using {gtsummary}
+## Create a contingency table:
+counts_table <-
+  penguins_data |>
+  group_by(species, island, sex) |>
+  count() |>
+  ungroup()
```

Comment:
I'd rather not add these examples. People have different ways of doing their stuff in R (and may not know all the tidyverse conventions); I just wanted to give the {gtsummary} to show people a nice, straightforward way to create publication-ready tables. Apart from that, I'd prefer to give them the chance to make their own changes at will (of course I'm 200% into {tidyverse} myself 😄). What do you think?

Reply:
Ok! No worries! Can revert changes then

Reply:
We'll see if we have time haha

Reply:
It's gonna be hard but... keep it anyway, we'll suggest them to make edits, so if it's just copy-pasting into their script from the chat it shouldn't take long. I like the idea that they do it "at will" in this case, to gain confidence in making their own changes 😉